src/report_generator.cpp

```diff
@@ -388,6 +408,12 @@ R"(<table>
    out << "<p>" << build_timestamp << "</p>";
 }
 
+std::string prune_title_tags(const std::string& title){
```

Comment:
I can rewrite this without `regex` if we don't want to take on that dependency (which is notoriously broken in GCC <4.9).